packages/os/hachiko.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description= Waits for host to be in-service before running kubelet if autoscaling.should_wait is true.
+After=network-online.target settings-applier.service
+Before=kubelet.service
+Requires=settings-applier.service network-online.target
+# We only want to run once, at first boot. This file is created by hachiko
+# after a successful run.
+ConditionPathExists=!/var/lib/bottlerocket/hachiko.ran
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/hachiko
+RemainAfterExit=true
+
+[Install]
+WantedBy=kubelet.service
```

Comment:
I looked at all the other services in Bottlerocket and none of them use `*.service` here. Thus, I think a target should be used, and for this case `multi-user.target` should be adequate.